DEVELOPMENT.md

```diff
@@ -808,7 +718,37 @@ We use [Cypress](https://cypress.io) for running frontend-based end-to-end tests
 
 #### Running frontend end-to-end tests locally
 
-Make sure to [build the frontend assets](#building-the-frontend) and run `tools/bin/mage dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs. To run the stack when working on end-to-end tests, use the `tools/bin/mage dev:startDevStack` command. This will run the runs The Things Stack with proper configuration for the end-to-end tests. Note: this command does not output anything, but logs are written to `.cache/devStack.log`.
+The preferred way to run the end-to-end tests is to use the interactive launcher tool with:
+
+```bash
+$  tools/bin/mage dev:serveDevWebui
+```
+
+1. Make sure to run `tools/bin/mage dev:dbStop dev:dbErase dev:dbCreate dev:dbStart dev:initStack dev:sqlDump` to create a seed database which the tests can reset the database to in between runs.
+2. Run `tools/bin/mage dev:serveDevWebui`
+3. Select "Cypress" in the launcher tool
+4. Add the following environment variables:
+
+```bash
+  export NODE_ENV="development"
+  export CYPRESS_BASE_URL="http://localhost:8080"
+```
```

Comment:
This should be done in a separate terminal after the `serveDevWebui` is finished with its setup, right?

Or this necessary on the same terminal which ran the `tools/bin/mage dev:serveDevWebui` as well?

Reply:
> This should be done in a separate terminal after the serveDevWebui is finished with its setup, right?

Correct.